api-docs/2-Getting Started.md

```diff
@@ -128,9 +128,9 @@ This example publishes a `custom` asset.
     </build>
 ```
 
-##### Requirements:
```

Comment:
what about replacing it with `Consideration`? or something like `Common error`? 🤔

Reply:
Good idea. How about "Troubleshooting"?

Reply:
I like it